src/editor.js

```diff
@@ -2836,9 +2838,32 @@ config.defineOptions(Editor.prototype, "editor", {
         initialValue: true
     },
     readOnly: {
-        set: function(readOnly) {
+        set: function(/**@type{boolean}*/readOnly) {
             this.textInput.setReadOnly(readOnly);
             this.$resetCursorStyle();
+            if (!this.readOnlyCallback) {
+                this.readOnlyCallback = () => {
+                    if (!this.hoverTooltip) {
+                        this.hoverTooltip = new HoverTooltip();
+                    }
+                    var domNode = dom.createElement("div");
+                    domNode.innerText = "Editing is disabled";
```

Comment:
We should use the `nls` utility to allow integrators to translate this string into whatever locale they want.

Reply:
Right, totally forgot about it, added nls message and covered by tests